e2e/protractor-sauce.conf.js

```diff
@@ -0,0 +1,76 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter } = require('jasmine-spec-reporter');
+const puppeteer = require('puppeteer');
+
+const isHeadlessModeEnabled = true;
+
+const baseUrl = (process.env.TEST_URL || 'http://localhost:3451/').replace('https', 'http');
+
+exports.config = {
+    sauceUser: process.env.SAUCE_USERNAME,
+    sauceKey: process.env.SAUCE_ACCESS_KEY,
+    sauceSeleniumAddress: 'ondemand.saucelabs.com:443/wd/hub',
+    allScriptsTimeout: 111000,
+    suites: {
+      e2e: './**/*.e2e-spec.ts',
+      smoke: '../smoke-test/*.smoke-spec.ts'
+    },
+    multiCapabilities: [{
+           'browserName': 'internet explorer',
+           'platform': 'Windows 10',
+           'version': '11.103',
+           'name': 'snl-IE-tests',
+           'tunnel-identifier': 'reformtunnel',
+           'extendedDebugging': true,
+           'shardTestFiles': true,
+           'maxInstances': 2
+        }],
+    baseUrl: baseUrl,
+    framework: 'jasmine',
+    jasmineNodeOpts: {
+        showColors: true,
+        defaultTimeoutInterval: 130000,
+        isVerbose: true,
+        includeStackTrace: true,
+        print: function () {}
+    },
+    plugins: [{
+        package: 'protractor-screenshoter-plugin',
+        screenshotPath: './functional-output/crossbrowser/reports',
```

Comment:
currently, it's `./functional-output/e2e/screenshots` should we adjust or this is intentionally outside of e2e folder?

Reply:
yes its intentional as this pipeline script, looks to archive the reports at this location
https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/contino/YarnBuilder.groovy#L53